libs/blocks/locui/actions/view.js

```diff
@@ -39,6 +40,20 @@ export default function Actions() {
     `;
   }
 
+  if (!languages.value || languages.value.length === 0) {
```

Comment:
You could make the statement a little easier:
```suggestion
  if (!languages?.value?.length) {
```